<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttendanceWin</class>
 <widget class="QMainWindow" name="AttendanceWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AttendanceWin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="2,19">
      <item>
       <widget class="QFrame" name="frame">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(4, 131, 212);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>77</x>
           <y>7</y>
           <width>271</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>门禁管理信息平台</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>7</y>
           <width>51</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/menjin.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="showWidget">
         <attribute name="title">
          <string>实时图像采集</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>471</width>
            <height>521</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="8,1">
           <item>
            <widget class="QLabel" name="picLb">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 255);
background-color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>当前时间：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="timeLabel">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">border: 1px solid black;border-radius:15px;</string>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="RegisterWin" name="registerWidget">
         <attribute name="title">
          <string>注册窗口</string>
         </attribute>
        </widget>
        <widget class="SelectWin" name="selectWidget">
         <attribute name="title">
          <string>信息查询</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RegisterWin</class>
   <extends>QWidget</extends>
   <header>registerwin.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SelectWin</class>
   <extends>QWidget</extends>
   <header location="global">selectwin.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
